scripts/push-to-assets-branch

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+rm -rf /tmp/tmp-assets
+mkdir -p /tmp/tmp-assets
+cp -r assets/ index.yaml index.html sha256sum/ /tmp/tmp-assets
+if [[ -d charts ]]; then
+  cp -r charts/ /tmp/tmp-assets
+fi
```

Comment:
We should check for the existence of all of these directories at the top of the file:
```shell
if ! [[ -d assets && -d sha256sum && -d charts ]]; then
    echo "Cannot find required assets to push to assets branch: assets/ sha256sum/ charts/"
    exit 1
fi
```

Is there any reason why the `charts/` directory should be treated specially? iirc we always want `assets/` and `charts/` in sync, so you shouldn't be able to push assets without modifying both.

Reply:
added.